runtime/near-vm-logic/src/config.rs

```diff
@@ -541,4 +577,62 @@ impl ExtCosts {
             validator_total_stake_base => config.validator_total_stake_base,
         }
     }
+
+    pub fn count() -> usize {
+        ExtCosts::validator_total_stake_base as usize + 1
+    }
+
+    pub fn name_of(index: usize) -> &'static str {
```

Comment:
Let's implement `Display` trait instead.

Reply:
I shall implement `Display` for which class? Note, that we have `usize`, not enum as an input, and `usize` ->  `ExtCosts` conversion is not straightforward.

Reply:
> and usize -> ExtCosts conversion is not straightforward.

Why not? It should be a straightforward cast.

Reply:
int cannot be casted to enum, as far, as I could see.